sklearn/base.py

```diff
@@ -6,15 +6,37 @@
 import copy
 import warnings
 from collections import defaultdict
+import struct
 
 import numpy as np
 from scipy import sparse
 from .externals import six
 from .utils.fixes import signature
 from . import __version__
 
+_DEFAULT_TAGS = {
+    'deterministic': True,
+    'requires_positive_data': False,
+    'input_types': ['2darray'],
+    'test_predictions': True,
+    'input_validation': True,
+    'multioutput': False,
+    "missing_values": False,
+    'stateless': False,
+    'multilabel': False,
+    '_skip_test': False,
+    'multioutput_only': False}
```

Comment:
binary only would be an important tag for external libraries (and came up in the context of the GP here).

Reply:
Make sure you're clear that it's binary targets, not fratures

Reply:
Binary only is also relevant for calibration methods.